google-cloud-examples/src/main/java/com/google/cloud/examples/compute/deprecated/ComputeExample.java

```diff
@@ -1,2539 +0,0 @@
-/*
```

Comment:
Deleting this old "deperecated" example. This file is not compatible with google-cloud-compute 1.X client library.